src/blocks/send_block.rs

```diff
@@ -38,7 +38,7 @@ pub struct SendBlock {
 }
 
 impl SendBlock {
-    pub const LEN: usize = 152;
+    pub const LEN: usize = 32+32+16+64+8;
```

Comment:
This is never going to change, true, but I think it's clearer like this, each field its own number.

Reply:
Good idea!